05A-foundational-skills-DRAFT.Rmd

```diff
@@ -41,24 +41,471 @@ Make an illustration that shows how these are related, highlighting where in the
     + How do I explore them? (CRAN, vignettes, Help docs, blog posts)
     + When should I write my own?
 
-* Data (nest in DS workflow) - embed chaining functions
-    + How do I import it?
-    + How do I explore it? 
-    + How do I tidy it? 
-    + How do I transform it?
-    + How do I visualize it?
-    + How do I model it?
-    + How do I communicate it?
-    + Do data structures matter?
-    + Do data types matter?
-    + Tidy data
-
 * Functions
     + What are they?
     + Why do I need them?
     + How do I know which arguments to provide?
     + How are they related to packages?
     + When should I write my own?
     + How do I write my own?
+
+* Data 
+    + How does data integrate with packages and functions within a Project?
+
+## Chapter overview
+This chapter is designed to give you the skills and knowledge necessary to _get started_ in any of the walk through chapters. Our goal is to get you working with R using the RStudio *I*ntegrated *D*evelopment *E*nvironment (IDE) through a series of applied examples.  
+
+Each walkthrough chapter also has its own supplemental *Foundational Skills* section, which is intended to bridge the gap between material covered in this chapter and the material covered within a specific walk through.  
+
+Please note that this chapter is not intended to be a full and complete introduction to programming with R, nor for using R for data science. There are many excellent resources available which provide this kind of instruction, and we've listed them for you on page [AAA] in the **Resources** section [link for bookdown version].  
+
+## Foundational Skills Framework
+No two data science projects are the same, and rather than be overly prescriptive, this chapter errs on the side of creating a general framework for you to use as a  home base as you work through this text. The four basic concepts we will use to build our framework are:  
+
+* **Data**
```

Comment:
Wonder if these become your first level headings, then all other sections are second level headings that support one of data, projects, packages, and functions?

Reply:
yes, that makes sense!